packages/monaco/src/browser/monaco-quick-input-service.ts

```diff
@@ -188,10 +192,34 @@ export class MonacoQuickInputImplementation implements IQuickInputService {
             createList: <T>(
                 user: string, container: HTMLElement, delegate: IListVirtualDelegate<T>, renderers: IListRenderer<T, unknown>[], listOptions: IListOptions<T>
             ): List<T> => this.quickInputList = new List(user, container, delegate, renderers, listOptions),
-            styles,
         };
         return options;
     }
+
+    private getStyles(): IQuickInputStyles {
+        return {
+            widget: {},
+            list: {},
+            inputBox: {
+                inputBackground: this.colorRegistry.getColor('inputBackground'),
+                inputBorder: this.colorRegistry.getColor('inputBorder'),
+                inputForeground: this.colorRegistry.getColor('inputForeground'),
+                inputValidationErrorBackground: this.colorRegistry.getColor('inputValidation.errorBackground'),
+                inputValidationErrorBorder: this.colorRegistry.getColor('inputValidation.errorBorder'),
+                inputValidationErrorForeground: this.colorRegistry.getColor('inputValidation.errorForeground'),
+                inputValidationWarningBackground: this.colorRegistry.getColor('inputValidation.warningBackground'),
+                inputValidationWarningBorder: this.colorRegistry.getColor('inputValidation.warningBorder'),
+                inputValidationWarningForeground: this.colorRegistry.getColor('inputValidation.warningForeground'),
+                inputValidationInfoBackground: this.colorRegistry.getColor('inputValidation.infoBackground'),
+                inputValidationInfoBorder: this.colorRegistry.getColor('inputValidation.infoBorder'),
+                inputValidationInfoForeground: this.colorRegistry.getColor('inputValidation.infoForeground')
+            },
+            countBadge: {},
+            button: {},
+            progressBar: {},
+            keybindingLabel: {},
+        };
+    }
```

Comment:
It would be helpful to future monaco uplifters to point to the location in VSCode where these styles are calculated to make it easier to keep up to date. It would be even more helpful to fill out the other subfields :-).